core-bundle/src/Command/CrawlCommand.php

```diff
@@ -218,8 +218,7 @@ public function needsContent(CrawlUri $crawlUri, ResponseInterface $response, Ch
 
             public function onLastChunk(CrawlUri $crawlUri, ResponseInterface $response, ChunkInterface $chunk): void
             {
-                // We only update the message here, otherwise too many nonsense URIs will be shown
-                $this->progressBar->setMessage((string) $crawlUri->getUri(), 'title');
+                // noop
```

Comment:
Why not remove the method altogether?

Reply:
It's in the interface?

Reply:
Ah, you are right. I missed the `return new class($progressBar) implements …`.